### PR TITLE
Tweaks to support development in a Windows environment

### PR DESCRIPTION
**Pull request by HerbCaudill**
This PR adds an icon in `.ico` format and a configuration setting to allow building and developing Trellis in a Windows environment. 

I've also added a couple of Windows-specific notes to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,34 +33,55 @@ We designed Trellis as an experiment to test and demonstrate the functionality o
 
 To our surprise, MPL was reasonably performant at our scale and we were able to design a developer interface that is intuitive, familiar to developers using modern frontend libraries, and does not require developers to understand or interface with CRDTs. The end-user experience was also impressively free of data corruption: as long as data updates are done with as small of a "granularity" as possible there are very few instance of data merging in unexpected ways and users always converged to the same application state. The remaining challenges at the conclusion of this project were mostly in building a robust and decentralized networking layer for CRDT synchronization.
 
-## Setup 
+## Setup
+
+You can download the latest version of Trellis [from the releases page](https://github.com/automerge/trellis/releases).
 
-You can download the latest version of Trellis for Mac OSX here: https://github.com/automerge/trellis/releases
```
Comment (HerbCaudill) [inline README.md]: I noticed that [v0.1.1](https://github.com/automerge/trellis/releases/tag/v0.1.1) includes a win32 build, while the other releases only have the darwin build. Not sure how this is set up, would be nice if releases always included a Windows download.

Comment (HerbCaudill) [reply README.md]: Hi! I'm thinking about using CRDTs in a new project, and am playing with making some simple improvements to Trellis as a way of understanding Automerge and related technologies. 

I'm happy to tinker away on my own fork - let me know if there's any interest in PRs addressing some of the issues still open.

Comment (pvh) [reply README.md]: I'd highly recommend you check out some of our more recent work, then! In particular, you might want to start with https://github.com/inkandswitch/pushpin/, which is two generations later. I should have something even "fresher" to share in a few more days but I'm happy to give you a sneak peek if you want to see our latest stuff, it's just a bit too raw to share without having a little onboarding tutorial.

Comment (HerbCaudill) [reply README.md]: Cool, not sure how I missed this! Playing with it now.

**Pull request merged**